github/resource_github_repository_collaborators.go

```diff
@@ -444,6 +445,10 @@ func matchTeamCollaborators(repoName string, want []any, has []teamCollaborator,
 	for _, t := range want {
 		teamData := t.(map[string]any)
 		teamIDString := teamData["team_id"].(string)
+		teamSlug, err := getTeamSlug(teamIDString, meta)
```

Comment:
I think this might be left over from some removed changes?

Reply:
Yeah, great catch... it was missed when I was making changes after your comment.

Reply:
Yeah, still looks like it needs removing.